internal/consensus/metrics.go

```diff
@@ -122,6 +122,12 @@ type Metrics struct {
 	// correspond to earlier heights and rounds than this node is currently
 	// in.
 	LateVotes metrics.Counter `metrics_labels:"vote_type"`
```

Comment:
At this point (roughly) I see two hunks in the original commit (#7709) that have been discarded during the forward-port. Looking at those changes I see they are adding `ProposalTimestampDifference` to two constructors. IIRC these constructors are now autogenerated by "metrics.gen" aren't they? Hence the changes in the "make metrics" commit.

(just double-checking here)

Reply:
Yes, most of the original changes in this file (`metrics.go`) are now at the automatically generated (`metrics.gen.go`). It is maybe useful to compare the current autogenerated file content to the manually generate metrics setup on the original commit (I've checked, they match, but please double check).

Reply:
I did check 👍